index.js

```diff
@@ -28,9 +28,10 @@ function indentResolve(str, options) {
     return str;
 }
 
-module.exports = function(plugins, options) {
+module.exports = function(plugins, options, filterType) {
```

Comment:
I haven't made the change yet to support regexp. If you want this, let me know and I'll ad it in, just not within `options`.